src/ocamlorg_frontend/pages/academic_users.eml

```diff
@@ -115,6 +122,12 @@ Layout.render
           <a class="btn btn-lg btn-tertiary" href="<%s Url.industrial_users %>">Success Stories</a>
         </div>
       </div>
+ 
+
     </div>
+   
+ <div class="prose max-w-none text-white pl-6 pb-6">
+    If you want to contribute to a new success story, check out this <a href="https://github.com/ocaml/ocaml.org/blob/main/CONTRIBUTING.md#story" target="_blank" style="color: white !important; text-decoration: none;"><strong>contributing guide</strong></a> on GitHub.
+</div>
```

Comment:
I would move this to the industrial users page, since this is where the actual success stories are listed.